libkbfs/interfaces.go

```diff
@@ -107,11 +107,10 @@ type KBFSOps interface {
 	// isn't favorited.
 	DeleteFavorite(ctx context.Context, fav Favorite) error
 
-	// GetOrInitializeNewMDMaster gets the existing MD or initialize a new MD for
-	// master branch
-	GetOrInitializeNewMDMaster(ctx context.Context, mdops MDOps, h *TlfHandle) (
-		initialized bool, md ImmutableRootMetadata,
-		id TlfID, err error)
+	// GetTLFCryptKeys gets crypt key of all generations as well as
+	// TLF ID for tlfHandle.
```

Comment:
Mention that the keys are ordered, starting with the key for `FirstValidKeyGen`.